source/wp-content/themes/wporg-developer-2023/parts/front-page-header.html

```diff
@@ -0,0 +1,21 @@
+<!-- wp:wporg/global-header /-->
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"16px","right":"var:preset|spacing|edge-space","bottom":"16px","left":"var:preset|spacing|edge-space"}},"elements":{"link":{"color":{"text":"var:preset|color|white"}}}},"backgroundColor":"charcoal-2","textColor":"white","className":"is-sticky wporg-front-page-breadcrumb","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull is-sticky wporg-front-page-breadcrumb has-white-color has-charcoal-2-background-color has-text-color has-background has-link-color" style="padding-top:16px;padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:16px;padding-left:var(--wp--preset--spacing--edge-space)"><!-- wp:group {"align":"full","layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"}} -->
+	<div class="wp-block-group alignfull"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+	<div class="wp-block-group"><!-- wp:heading {"className":"is-style-default","fontSize":"small","fontFamily":"inter"} -->
+	<h2 class="wp-block-heading is-style-default has-inter-font-family has-small-font-size">Developers</h2>
+	<!-- /wp:heading --></div>
+	<!-- /wp:group -->
+	
+	<!-- wp:navigation {"textColor":"white","backgroundColor":"charcoal-2","className":"is-style-dropdown-on-mobile","style":{"spacing":{"blockGap":"24px"}},"fontSize":"small"} -->
+	<!-- wp:navigation-link {"label":"Blog","url":"https://developer.wordpress.org/news/","kind":"custom","isTopLevelLink":true} /-->
+	
+	<!-- wp:navigation-link {"label":"Forums","url":"https://wordpress.org/support/forums/","kind":"custom","isTopLevelLink":true} /-->
+	
+	<!-- wp:navigation-link {"label":"Documentation","type":"custom","url":"https://wordpress.org/support/","kind":"custom","isTopLevelLink":true} /-->
+	
+	<!-- wp:navigation-link {"label":"Get Involved","type":"custom","url":"https://make.wordpress.org/support/handbook/","kind":"custom","isTopLevelLink":true} /-->
+	<!-- /wp:navigation --></div>
+	<!-- /wp:group --></div>
+	<!-- /wp:group -->
```

Comment:
The indentation is strange in this file — it doesn't seem to affect anything, but it would be nice to fix.

Reply:
This is the indentation when you "copy block" from Gutenberg. I'm always scared to touch the exported template code. I always create a whole bunch of validation errors. ☺️ 

I'll remove it.